src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -52,19 +54,90 @@ private void ProcessResults()
         {
             CSVMetricReader reader = new CSVMetricReader(Configuration.FileLogPath);
 
-            foreach(string testCaseName in reader.TestCases)
+            foreach (string testCaseName in reader.TestCases)
```

Comment:
Can you remove the dumping of raw data to the console?  We don't need it now with this change.